test/pollNonce.js

```diff
@@ -1,10 +1,20 @@
 /* eslint-env mocha */
-/* global contract */
+/* global contract assert */
+
+const BigNumber = require('bignumber.js');
+const utils = require('./utils.js');
 
 contract('PLCRVoting', () => {
-  describe('Function: pollNonce', () => {
-    it('should initialize the pollNonce to zero');
-    it('should properly increment the pollNonce before a new poll is created');
```

Comment:
I'm not sure it is possible to test this value _before_ the poll is created? If not, then this test is already implemented in `startPoll.js`